### PR TITLE
MESH-1105/ Minor Weight Mgt

### PR DESCRIPTION
**Pull request by satyamakgec**

### CODE REVIEW

File: pallets/committee/src/lib.rs [hunk 1]
```diff
@@ -722,6 +722,11 @@ impl<
             r => Err(O::from(r)),
         })
     }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    fn successful_origin() -> O {
```
Comment (maxsam4) [inline pallets/committee/src/lib.rs:727]: Can we please avoid using `O` (Oh)? It is easy to confuse with `0`(Zero).

File: pallets/committee/src/lib.rs [hunk 2]
```diff
@@ -742,4 +747,9 @@ impl<
...
```
Comment (maxsam4) [inline pallets/committee/src/lib.rs:752]: O ~ 0

File: pallets/staking/src/testing_utils.rs [hunk 3]
```diff
@@ -41,23 +40,23 @@ pub fn get_minimum_balance<T: Trait>() -> Balance<T> {
...
+//     } else {
+//         vec![]
+//     };
+//     Identity::cdd_register_did(RawOrigin::Signed(1005).into(), account, None, signing_items);
+//     if let LinkedKeyInfo::Unique(did) = Identity::key_to_identity_ids(account).unwrap() {
+//         Ok(did)
+//     } else {
+//         return Err("Error in providing the IdentityId");
+//     }
+// }
```
Comment (maxsam4) [inline pallets/staking/src/testing_utils.rs:59]: Remove dead code?

File: pallets/staking/src/testing_utils.rs [hunk 4]
```diff
@@ -80,8 +79,8 @@ pub fn create_stash_controller<T: Trait>(
 ) -> Result<(T::AccountId, T::AccountId), &'static str> {
     let stash = create_funded_user::<T>("stash", n, balance_factor);
     let controller = create_funded_user::<T>("controller", n, balance_factor);
-    let did = provide_did_to_user::<T>(stash, Some(controller))?;
-    print!("Print the DID : {:?}", did);
+    // let did = provide_did_to_user::<T>(stash, Some(controller))?;
+    // print!("Print the DID : {:?}", did);
```
Comment (maxsam4) [inline pallets/staking/src/testing_utils.rs:83]: Remove dead code?

File: pallets/committee/src/lib.rs (hunk 1 above)
Comment (satyamakgec) [reply pallets/committee/src/lib.rs:727]: We can avoid, But it is inherited from the substrate that's why I avoid changing them

**Pull request merged**